README.md

```diff
@@ -20,6 +20,12 @@ See [CONTRIBUTING](https://github.com/opiproject/opi/blob/main/CONTRIBUTING.md)
 
 ## Getting started
 
+a special transport should be created on IPU side by
```

Comment:
do you expect user to create this manually or gRPC handles it? I think some small clarification needed here